src/EnergyPlus/FluidProperties.cc

```diff
@@ -243,181 +218,397 @@ namespace FluidProperties {
 
         // For default "glycol" fluids of Water, Ethylene Glycol, and Propylene Glycol
 
-        static Array1D<Real64> const
-            DefaultGlycolTemps(DefaultNumGlyTemps, {-35.0, -30.0, -25.0, -20.0, -15.0, -10.0, -5.0,  0.0,   5.0,  10.0, 15.0, 20.0,
-                                                    25.0,  30.0,  35.0,  40.0,  45.0,  50.0,  55.0,  60.0,  65.0, 70.0, 75.0, 80.0,
-                                                    85.0,  90.0,  95.0,  100.0, 105.0, 110.0, 115.0, 120.0, 125.0}); // 33 total temperature points
+        constexpr std::array<Real64, DefaultNumGlyTemps> DefaultGlycolTemps = {
```

Comment:
`static Array1D<Real64> const` to `constexpr std::array<Real64, constant that shows size_of_array>`.